### DISCUSSION TITLE
Need some exact information regarding EPS. Injestion Rate,etc.

### DISCUSSION THREAD

**Discussion by bakliwalp** [Security-Onion-Solutions/securityonion#10278] [Unsupported Versions] [answered]
Hello Everyone. Hope you all are doing well.
I am planning to use Security Onion. Although I am completely beginner in SOC environment and setting up a SOC lab, so please understand me if I am not able to explain it clearly.

I wan to know 
1) How much Events per Second does Security Onion can handle?
2) What is the Log Injestion Rate it can handle?
3) If I want to create 100 users with different roles, is it possible? If no, what is the maximum limit?

Thank you in advance for any help.

**Comment by TOoSmOotH** [comment, marked as answer]
> 1. How much Events per Second does Security Onion can handle?

We have customers/users doing over 100k EPS. This is really determined by the hardware you are using. 

> 2. What is the Log Injestion Rate it can handle?

See # 1

> 3. If I want to create 100 users with different roles, is it possible? If no, what is the maximum limit?

There is no limit to the amount of roles in SOC. There could be a limit in Elastic Search but I am not sure.
